packages/payload/src/config/types.ts

```diff
@@ -977,6 +983,44 @@ export type Config = {
   email?: EmailAdapter | Promise<EmailAdapter>
   /** Custom REST endpoints */
   endpoints?: Endpoint[]
+  /**
+   * Options for folder view within the admin panel
+   */
+  folders?: {
+    /**
+     * An array of functions to be ran when the folder collection is initialized
+     * This allows plugins to modify the collection configuration
+     */
+    collectionOverrides?: (({
+      collection,
+    }: {
+      collection: CollectionConfig
+    }) => CollectionConfig | Promise<CollectionConfig>)[]
+    /**
+     * Ability to view hidden fields and collections related to folders
+     *
+     * @default false
+     */
+    debug?: boolean
+    /**
+     * Enable folders in the admin panel
+     *
+     * @default false
+     */
+    enabled?: boolean
```

Comment:
Is this still needed? To determine if folder view is enabled at the root you could check each collection for `admin.folders`. If at least one is found, you can assume it's enabled and add the folder collection.

Reply:
This is how query presets works as well.